packages/web-client/app/components/card-pay/withdrawal-workflow/transaction-status/index.hbs

```diff
@@ -0,0 +1,47 @@
+<ActionCardContainer
+  @header="Token bridge"
+  @isComplete={{@isComplete}}
+  data-test-withdrawal-transaction-confirmed-is-complete={{@isComplete}}
+>
+  <ActionCardContainer::Section
+    @title={{concat "Bridging tokens to " (network-display-info "layer1" "fullName")}}
+    @dataTestId="withdrawal-confirmed"
+  >
+    <Boxel::ProgressSteps
+      class="transaction-status__steps"
+      @progressSteps={{this.progressSteps}}
+      @completedCount={{this.completedCount}}
+      as |progressStep|
+    >
+      <span
+        class="transaction-status__step-title"
+        data-test-token-bridge-step={{progressStep.index}}
+        data-test-completed={{progressStep.completed}}
+      >
+        {{progressStep.title}}
+      </span>
+
+      <span class="transaction-status__step-status">
+        {{#if (eq progressStep.index 0)}}
+          <Boxel::Button @as="anchor" @size="extra-small" @href={{this.blockscoutUrl}} target="_blank" rel="noopener" data-test-blockscout-button>
+            View on Blockscout
+          </Boxel::Button>
+        {{/if}}
+        {{#if (eq progressStep.index 1)}}
+          <Boxel::Button @as="anchor" @size="extra-small" @href={{this.bridgeExplorerUrl}} target="_blank" rel="noopener" data-test-bridge-explorer-button>
+            View in Bridge Explorer
+          </Boxel::Button>
+        {{/if}}
+        {{#if (eq progressStep.index 2)}}
+          {{#if progressStep.completed}}
```

Comment:
I think we ideally want to show this etherscan button as soon as we have `withdrawalTxnViewerUrl`, even if the operation is not yet complete.

Reply:
Had a look, there is a [task in Linear for claiming tokens](https://linear.app/cardstack/issue/CS-1234/withdrawal-claim-card). This task has a proposal for the addition of a claim card that replaces step 3. There is also a note in the original task about this.